Source/Krypton Components/Krypton.Navigator/View Draw/ViewDrawNavCheckButtonBase.cs

```diff
@@ -114,9 +114,9 @@ internal abstract class ViewDrawNavCheckButtonBase : ViewDrawButton,
             : base(stateDisabled, stateNormal, stateTracking,
                    statePressed, null, null, orientation, true)
         {
-            Debug.Assert(navigator != null);
+            Debug.Assert(navigator is not null);
 
-            Navigator = navigator;
+            Navigator = navigator ?? throw new ArgumentNullException(nameof(navigator));
```

Comment:
I prefer this style to the `if( ## is null)` usage

Reply:
Only applies of course if there's an assignment in the code. 
Mostly done in this way. Will take this further.